### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by iuliancorcoja**
# Change or Enhancement

#### Changelog

**Changed**

- Readme documentation updated.

**Removed**

- Cleaned up unused Maven dependencies.

#### Reviewing

@tlawrie, @marcusdroy, it would be good to add Listener Service architecture image to the documentation. How to reference it in the markdown though: add URL from https://www.useboomerang.io/docs or create a folder in this repo where all documentation assets will be stored?

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,42 +38,13 @@
...
 			</exclusions>
 		</dependency>
-		<dependency>
-			<groupId>com.fasterxml.jackson.core</groupId>
-			<artifactId>jackson-databind</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>com.fasterxml.jackson.core</groupId>
-			<artifactId>jackson-core</artifactId>
-		</dependency>
```
Comment (tlawrie) [inline pom.xml:52]: Hey @iuliancorcoja why dont we need these ones anymore? i thought we used Jackson pretty heavily in our services?

File: pom.xml [hunk 2]
```diff
@@ -42,42 +38,13 @@
...
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-log4j2</artifactId>
+			<artifactId>spring-boot-starter-web</artifactId>
```
Comment (tlawrie) [inline pom.xml:43]: By having all of web, does this bring in more / less dependencies than have the particular spring-core / spring-web / spring-context?

**Review by tlawrie — COMMENTED** [review]
@iuliancorcoja great changes to the README! Thank you.

A few clarification questions for my own knowledge and then this can most likely be merged in.

**Comment by iuliancorcoja** [comment]
We do use it pretty heavily. Fasterxml’s jackson core is a dependency of Spring Boot, thus no need to include it twice unless a specific version is needed (which isn’t the case in this project).

See here:
https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-dependency-versions.html

**Comment by iuliancorcoja** [comment]
The oddities of ‘diff’! So spring boot web was in ‘pom.xml’ before, I just deleted log4j for the same reason as jackson core – already included in Spring Boot.

**Pull request merged**